guides/v2.2/cloud/project/new-relic.md

```diff
@@ -137,27 +156,103 @@ Using the New Relic dashboard, you can immediately track and find:
+
+-  [Apdex score](https://docs.newrelic.com/docs/apm/new-relic-apm/apdex/apdex-measure-user-satisfaction)
+-  error rate
+-  disk space on your Production environment
+
+These policies set thresholds for warning and critical conditions that affect performance based on industry best practices. When your site experiences infrastructure or application issues that trigger an alert threshold, New Relic sends alert notifications so you can proactively address the issue. To use these policies, you must configure notification channels to receive the alert messages.
+
+{:.bs-callout-info}
+For Pro Staging and Integration environments and Starter environments, use [Health notifications]({{ page.baseurl }}/cloud/integrations/health-notifications.html) to monitor disk space.
+
+**Prerequisites:**
+
+-  Credentials to log in to the [New Relic account](#manage-your-new-relic-account) for your Cloud project.
+-  Verify that your Cloud environment is [connected to New Relic](#connect-to-new-relic).
+
+### Review Adobe-generated alert policies
+
+1. Use your Project Owner New Relic credentials to [log in to New Relic](https://login.newrelic.com/login).
+
+1. From the navigation menu, select **Alerts** > **Alert Policies**.
+
+1. In the Search Policies field, search for _Adobe Generated Policy_.
+
+1. You should see a set of policies like the ones below:
+
+   ![Generated alert policies]({{ site.baseurl }}/common/images/cloud/cloud-newrelic-alert-policies.png){:width="600px"}
+
+   {:.bs-callout-info}
+   If you do not see these alert policies, submit a Magento support ticket. Include your project ID in the ticket. These policies are available only on the Pro Production environment.
+
+1. Click an alert policy name to review the conditions that specify the conditions that trigger alerts.
 
-### New Relic and Blackfire
+### Configure notification channels
+
+To use the Adobe-generated alert policies to monitor your Production site, you must configure notification channels and map them to alert policies. Notifications about performance issues go out to all channels associated with an alert policy when conditions on the application or infrastructure trigger an alert. You also receive notifications when an issue is acknowledged and closed.
+
+New Relic provides templates for configuring different types of notification channels including email, Slack, PagerDuty, webhooks, and more. See the [Instructions for specific notification channels](https://docs.newrelic.com/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-control-where-send-alerts#channel-types) in the New Relic documentation to review the prerequisites for using each type.
+
+The following instructions describe the high level steps to configure a Slack channel to receive New Relic alerts when the available disk space on your {{ site.data.var.ee }} Production server exceeds the low disk space threshold specified in the _Adobe Generated Disk Space Policy_ alert policy.
```

Comment:
Seems like this belongs where the other Slack integrations are. Again, just thinking aloud. nothing to do here.

Reply:
Yes-- I thought so too. In a separate PR, I have updates to the Health notifications topic that point to the Adobe-generated policies section in this topic.